Instructions/06-design-report-in-power-bi-desktop.md

```diff
@@ -9,579 +9,400 @@ lab:
+In this task, you'll design the first report page. When you’ve completed the design, the page will look like the following:
 
-15. In the **Save As** window, navigate to the **D:\PL300\MySolution** folder.
+![Image of page 1, comprising a logo, two slicers, and three visuals.](Linked_image_Files/06-finished-report-page.png)
 
-16. Click **Save**.
+1. In Power BI Desktop, to rename the page, at the bottom-left, right-click **Page 1**, then **Rename** the page as **Overview**.
 
-	![Picture 4](Linked_image_Files/07-design-report-in-power-bi-desktop_image11.png)
+    *Tip: You can also double-click the page name to rename it.*
 
-### **Task 4: Design page 1**
+1. To add an image, on the **Insert** ribbon tab, from inside the **Elements** group, select **Image**.
 
-In this task you will design the first report page. When you’ve completed the design, the page will look like the following:
+    ![Picture 1](Linked_image_Files/07-design-report-in-power-bi-desktop_image15.png)
 
-![Image of page 1, comprising a logo, two slicers, and three visuals.](Linked_image_Files/07-design-report-in-power-bi-desktop_image12.png)
+1. In the **Open** window, navigate to the **D:\PL300\Resources** folder.
 
-1. In Power BI Desktop, to rename the page, at the bottom-left, right-click **Page 1**, and then select **Rename**.
+1. Select the **AdventureWorksLogo.jpg** file, and then select **Open**.
 
-	![Picture 36](Linked_image_Files/07-design-report-in-power-bi-desktop_image13.png)
+1. Drag the image to position it at the top-left corner, and also drag the guide markers to resize it.
 
-	*Tip: You can also double-click the page name to rename it.*
+     ![Picture 12](Linked_image_Files/07-design-report-in-power-bi-desktop_image17.png)
 
-2. Rename the page as **Overview**, and then press **Enter**.
+1. To add a slicer, first de-select the image by clicking an empty area of the report page, then select the **Slicer** in the **Visualizations** pane.
 
-	![Picture 37](Linked_image_Files/07-design-report-in-power-bi-desktop_image14.png)
+     ![Picture 49](Linked_image_Files/07-design-report-in-power-bi-desktop_image18.png)
 
-3. To add an image, on the **Insert** ribbon tab, from inside the **Elements** group, click **Image**.
+1. In the **Fields** pane, drag the **Date \| Year** field (not the **Year** level of the hierarchy) into the slicer **Field** in Visualizations pane.
+    1. *The labs use a shorthand notation to reference a field. It will look like this: **Date \| Year**. In this example, **Date** is the table name and **Year** is the field name.*
 
-	![Picture 1](Linked_image_Files/07-design-report-in-power-bi-desktop_image15.png)
+1. To convert the slicer from a list to a dropdown, at the top-right of the slicer, select the down-arrow, and then select **Dropdown**.
 
-4. In the **Open** window, navigate to the **D:\PL300\Resources** folder.
+    ![Slicer style](Linked_image_Files\06_slicer_style.png)
```

Comment:
Photo is missing
![image](https://user-images.githubusercontent.com/110993710/209653932-2ec80c5b-29c7-40f9-8c36-83ff269e3c62.png)